report_generator/input.json

```diff
@@ -0,0 +1,136 @@
+{
+  "repositoryLink": "https://repository.byont/",
+  "commit": "j230a-al4uf-zmh2",
+  "contractPath": "https://contract-path-url.byont/",
+  "solcVersion": "2.04",
+  "foundryVersion": "0.85",
+  "slitherVersion": "1.45",
+  "mythrillVersion": "1.77",
+  "smtcheckerVersion": "0.8.17",
+  "tools": [
+    {
+      "name": "Slither",
+      "nrOfIssues": 42069,
+      "version": "1.54.33",
+      "hasDetectors": true,
+      "detectors": [
+        {
+          "name": "reentrancy-eth",
+          "impact": "High",
+          "confidence": "Medium",
+          "function": "there should some sort of a code snapshot",
+          "linesOfRelevance": [
+            "there also should be some sort of a code snapshot, but apparently it is usually a bit smaller than the ones for Functions",
```

Comment:
Where'd you get this input from? Nice for a demo to start with. But would be even sicker if we can get the output data of a real contract. For instance, one of Enzo's analyzed (and vulnerable) contracts.

Reply:
This will be an array of line numbers, since we have the commit (and after your suggestion also the sha of the file), we know that the source code lines are correct. This saves file size.